x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.tsx

```diff
@@ -163,6 +167,8 @@ export const RulesList: React.FunctionComponent = () => {
   );
   const [showErrors, setShowErrors] = useState(false);
 
+  const isRuleStatusFilterEnabled = getIsExperimentalFeatureEnabled('ruleStatusFilter');
```

Comment:
@JiaweiWu Should we also check in observability if the experimental feature is enabled? In this case, could you export `getIsExperimentalFeatureEnabled` in this [file](https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/index.ts), so that we can reuse it?

Reply:
Ok I see, thanks for clarifying!